File: src/test/java/ListTest.java
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

public class ListTest extends Initialization {

    private MainNavBar mainNavBar = new MainNavBar();
    private SignInPage signInPage = new SignInPage();
    private YourListsPage yourListsPage = new YourListsPage();

    @BeforeEach
    public void setup() {
        mainNavBar.navigate();
        signInPage.loginWithValidData();
    }

    @Test
    public void checkList() {
        mainNavBar.clickOnUserButton();
        mainNavBar.clickOnYourLists();
        yourListsPage.clickOnMegunhatatlan();
        Assertions.assertTrue(yourListsPage.checkMovie());
    }
}

File: src/tests/list_test.py
import unittest

import ddt as ddt

from your_lists_page import YourListsPage
from initialization import Initialization

@ddt
class ListTest(Initialization):

    def setup(self):
        super().setUp()
        self.yourListsPage = YourListsPage(self.driver)
        self.yourListsPage.navigate_to()
        #signInPage.loginWithValidData()


if __name__ == "__main__":
    unittest.main()
